src/Umbraco.Infrastructure/Persistence/Repositories/Implement/ContentRepositoryBase.cs

```diff
@@ -1080,80 +1080,9 @@ protected virtual int GetReservedId(Guid uniqueId)
 
         #endregion
 
+        [Obsolete("This method is no longer used as the persistance of relations has been moved to a notification handler. Scheduled for removal in Umbraco 19.")]
```

Comment:
Making this a no-op method is probably fine, since it's a protected method and not called within our codebase anymore... But if we do want to make this fully backwards compatible, the implementation in `ContentRelationsUpdate` could be moved to an internal static method that requires all dependencies to be passed into it...

Otherwise, this could at least log a warning or error to indicate why no relations are persisted 😄

Reply:
Have added the logging - I think that will suffice.